### PR TITLE
Improve indentation

### PR DESCRIPTION
**Pull request by 3lvis**
From 1-10 I'm not sure how good is this.

### CODE REVIEW

File: Source/TutorialController.swift [hunk 1]
```diff
@@ -61,6 +68,7 @@ public extension UIViewController {
     for modelView in model.views() {
       self.view.addSubview(modelView as! UIView)
```
Comment (kostiakoval) [inline Source/TutorialController.swift]: still using `self` ?

**Comment by 3lvis** [comment]
@kostiakoval oh, this is old, will make a PR fixing it.

File: Pod/Demo/AppDelegate.swift [hunk 2]
```diff
@@ -11,61 +11,84 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
+      let page2 = UIViewController(model: model2)
+      let page3 = UIViewController(model: model3)
+      let page4 = UIViewController(model: model4)
+      let page5 = UIViewController(model: model5)
+
+      tutorialController.add([page1, page2, page3, page4, page5])
+
+      TutorialController.setFont(
+        UIFont(name: "ArialRoundedMTBold",
+          size: 42.0)!)
```
Comment (kostiakoval) [inline Pod/Demo/AppDelegate.swift]: broken `:` indent

Comment (zenangst) [inline Pod/Demo/AppDelegate.swift]: I don't think Swift indents on `:`.
It just adds 2 space indents for the newline.

File: Source/TutorialController.swift (hunk 1 above)
Comment (3lvis) [inline Source/TutorialController.swift]: Fixed on #18

File: Pod/Demo/AppDelegate.swift (hunk 2 above)
Comment (3lvis) [inline Pod/Demo/AppDelegate.swift]: Fixed on #18

**Pull request merged**